draft-ietf-rats-architecture.md

```diff
@@ -316,6 +316,26 @@ Relying Party:
 : A remote server that will securely grant the Attester
 permission to continue operating (i.e., not reboot) for a period of time
 
+## FIDO Biometric Authentication
+
+In the FIDO protocol, it is the device in the user’s hand that authenticates the human user by biometrics like fingerprints or PIN and password.
+The comparison of the input from the user to the reference or expected values is done by the device, often a mobile phone, not the relying party.
```

Comment:
So the device is a verifier?  (not sure why "not the relying party" is in the sentence)

Reply:
The Authenticator is a component of the Attester, isn't it? It is used (in the weakest sense) for evidence generation, I think.

Reply:
The device verifies biometric input locally. Nothing to do with verifying attestation evidence. Hopefully recent commit addresses this.